configuration/simple-topology-0.json

```diff
@@ -0,0 +1,15 @@
+{ "nodeAddress":
```

Comment:
Why is this needed? I thought that we agreed that local ip address should (optionally) be provided through  cli along with port number?